types/bool.go

```diff
@@ -0,0 +1,40 @@
+package types
+
+type (
+	Bool bool
+)
+
+var True = Bool(true)
```

Comment:
I think you can use:

```go
const True Bool = true
```

And you will export a const instead of a var =D

Example: https://play.golang.org/p/thbXIWqlJpO